### PR TITLE
Changed: default colors for calendars and color picker.

### PR DESCRIPTION
**Pull request by jsilvestre**

### CODE REVIEW

File: client/app/views/tags.coffee [hunk 1]
```diff
@@ -17,7 +15,7 @@ module.exports = class TagsView extends BaseView
         return this
 
     tagAdded: (ev, ui) ->
-        ui.tag.css 'background-color', colorHash ui.tagLabel
+        ui.tag.css 'background-color', ColorHash.getColoru(ui.tagLabel, 'cozy')
```
Comment (m4dz) [inline client/app/views/tags.coffee]: I thinks there's a typo here (`getColoru`), nope?

File: client/app/views/styles/_menu.styl [hunk 2]
```diff
@@ -145,15 +147,32 @@ ul#menu
...
+    ul.dropdown-menu li:first-of-type
+        &[data-picker-visible=true]:hover
+            background white
+
+            a:hover
+                background white
+
+
+    ul.color-picker
+        //display none
```
Comment (m4dz) [inline client/app/views/styles/_menu.styl]: Could you remove useless dead code?

File: client/app/views/tags.coffee (hunk 1 above)
Comment (jsilvestre) [inline client/app/views/tags.coffee]: I'm soon going to Asia, that's why I add it (if you don't understand why, I'll explain ;)). I fix it asap.

**Comment by jsilvestre** [comment]
Thank you for the feedback, I fixed what had to be fixed.

**Comment by m4dz** [comment]
OK, sounds good for me, thanks a lot!

**Pull request merged**